string_cases/lib/string_cases.ex

```diff
@@ -0,0 +1,41 @@
+defmodule StringCases do
+  @moduledoc """
+  A collection of functions for changing the casing of string values
+  """
+
+  @doc """
+  Given a string return a properly CamelCased string.
+  Spaces and punctuation should be removed and the first letter of each word capitalized.
+
+  # Examples
+
+  iex> StringCase.camel_case("Elixir-School is awesome!")
+  ElixirSchoolIsAwesome
+  """
+  def camel_case(input) do
+  end
+
+  @doc """
+  Snake_case the inputed string value.
+  The resulting string should not include any punctuation other than underscores.
+  Spaces are replaced with underscore and words are downcased.
+
+  # Examples
+
+  iex> StringCase.snake_case("Elixir-School is awesome!")
+  elixir_school_is_awesome
+  """
+  def snake_case(input) do
+  end
+
+  @doc """
+  Everyone's favorite!  Wavy case alternates upper- and downcased letters started with a capitalize letter.
```

Comment:
There is something wrong here with spaces and -, maybe try to find out what?

Reply:
@gemantzu I _think_ this is correct but I'm not an English major 😆 

I'm saying upper and lowercased  `upper-` being the prefix.  I can change this.